### PR TITLE
Add V2 Polling Lambda

### PR DESCRIPTION
**Pull request by FinnIckler**
Split from #10058. Depends on that being merged first because of the SQS queue, but I have already deployed all the infrastructure to test, so this works. 

Review needed for the ruby changes, i.e the hook and the lambda function

### CODE REVIEW

File: app/jobs/add_registration_job.rb [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 class AddRegistrationJob < ApplicationJob
   before_enqueue do |job|
-    Rails.cache.write("#{job.arguments[1]}-#{job.arguments[2]}-processing", true)
+    competition_id = job.arguments[1]
+    user_id = job.arguments[2]
```
Comment (gregorbg) [inline app/jobs/add_registration_job.rb:6]: Ruby 101: `competition_id, user_id = job.arguments`

Comment (FinnIckler) [reply app/jobs/add_registration_job.rb:6]: Ruby 102, arrays begin at 0 :P and I don't think `_,competition_id, user_id = job.arguments` looks that great?

Comment (FinnIckler) [reply app/jobs/add_registration_job.rb:6]: Alternatively I could change the order of the job params

Comment (gregorbg) [reply app/jobs/add_registration_job.rb:6]: Ruby 103: `competition_id, user_id = job.arguments.drop(1)`

Comment (gregorbg) [reply app/jobs/add_registration_job.rb:6]: And to be honest your underscore approach isn't too bad either.

Comment (FinnIckler) [reply app/jobs/add_registration_job.rb]: ok, if you fine with that, i changed it to that

**Review by gregorbg — APPROVED** [review]

**Pull request merged**